sdk/spring/spring-cloud-azure-starter-monitor-test/src/test/java/com/azure/SpringMonitorTest.java

```diff
@@ -105,6 +107,10 @@ public void applicationContextShouldOnlyContainTheAzureMetricExporter() {
 
   @Test
   public void shouldMonitor() throws InterruptedException, MalformedURLException {
+
+    // Only required with GraalVM native test execution
+    SpringMonitorTest.class.getResourceAsStream("/logback.xml");
```

Comment:
can you add a bit more comment about why here?

Reply:
The log telemetry did not work because the OTel Logback appender was not added to Logback. I have added this code line to have more insights. And it fixes the issue. I don't know why. It seems a Logback issue with GraalVM native.